src/App.tsx

```diff
@@ -8,11 +8,11 @@ import { useInitializeMap } from "./couldBeSharedComponents/Map/useInitializeMap
 
 function App() {
   const [isLeftPanelOpenOverride, setIsLeftPanelOpenOverride] = useState(true);
-  const [isLeftSubpanelOpen, setIsLeftSubpanelOpen] = useState(true);
+  const [isSubpanelOpen, setIsSubpanelOpen] = useState(true);
```

Comment:
I was thinking at some point there might be a right subpanel but was told it was unlikely. I figure if the day ever comes when we have a right subpanel we can rename vars then. For now, Im leaving it to be named with no left and right to cut wordiness and make it easier to differentiate from all the other state that has the words 'left' and 'right' in it.